deploy/eck-stack/examples/kibana/http-configuration.yaml

```
@@ -5,10 +5,11 @@ eck-kibana:
   count: 1
 
   # Reference to ECK-managed Elasticsearch resource, ideally from {{ "elasticsearch.fullname" }}
-  #
-  elasticsearchRef:
-    name: es-quickstart-eck-elasticsearch
-    # namespace: default
+  # Should be used only when deploying the individual eck-kibana chart or eck-stack without Elasticsearch
+  # Comment it out if you are deploying eck-kibana as part of the eck-stack chart with Elasticsearch, because the chart logic takes care of it
+  # elasticsearchRef:
+  #   name: es-quickstart-eck-elasticsearch
```

Nit: The example name `es-quickstart-eck-elasticsearch` is a bit confusing since when using eck-stack the ES resource is actually named `elasticsearch` (via `fullnameOverride`). 

Consider using a placeholder that's more instructive for standalone usage:

```suggestion
  #   name: <your-elasticsearch-resource-name>
```

Or if you want to keep a concrete example, perhaps `elasticsearch` to match the eck-stack default.